docusaurus/docs/reference/ha/overview.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_label: Overview
+sidebar_position: 05
+---
+
+# Controller HA 
+
+## Overview
+
+Ziti controllers can be run in a cluster for high availablity and performance scaling.
+
+:::warning
+
+**NOTE: Controller HA is still in Beta** 
+
+It's quite functional now, but we are continuing to test and refine before we mark it GA.
+:::
+
+
+### For SDK Clients/Tunnelers
+
+A controller cluster offers the following advantages:
+
+1. Horizontal scaling of SDK client services such as
+    1. Service lookups
+    1. Session creation
+1. Horizontal scaling of circuit creation
+
+This means that for everything that SDK clients and tunnelers depend on, controllers
+can be scaled up and placed strategically to meet user demand. 
+
+The following limitations currently apply:
+
+1. Circuits are owned by a controller. If the controller goes down, the circuit 
```

Comment:
are there issues to track when these limitations will be eliminated? I'd like to see some text about how these are going to be addressed in the future. ideally with a link they reader could use to track if they wanted. also, removing the doc about limitations should be tracked too. feels like the kinda thing that can be easily forgotten

Reply:
Added a link to the router next project